SteamReview/SteamReview/Steam.cpp

```diff
@@ -0,0 +1,320 @@
+#include "Steam.h"
+#include <iostream>
+#include <string>
+
+FSteam::FSteam()
+{
+	Uncategorized = FCategory("Uncategorized");
+	ActiveMenuCommand = EMenuCommand::MainMenu;
+};
+
+//---------------------------------------------------------
+void FSteam::OpenMainMenu()
+{
+	std::system("cls");
+	std::cout << "--------------------WELCOME TO STEAM--------------------\n\nWhat do you want to do?\n----------------------------------------" << std::endl;
+	std::cout << "1 - Add a new game" << std::endl;
+	std::cout << "2 - Category Manager" << std::endl;
+	std::cout << "3 - Display Games" << std::endl;
+	std::cout << "4 - Exit" << std::endl;
+	std::cout << "Please choose an option to follow..." << std::endl;
+
+	int UserInput;
+	std::cin >> UserInput;
+	InputValidate(UserInput, 1, 4);
+
+	switch (UserInput)
+	{
+	case 1:
+		ActiveMenuCommand = EMenuCommand::AddGame;
+		break;
+	case 2:
+		ActiveMenuCommand = EMenuCommand::CategoryManager;
+		break;
+	case 3:
+		ActiveMenuCommand = EMenuCommand::DisplayGames;
+		break;
+	case 4:
+		ActiveMenuCommand = EMenuCommand::Exit;
+		break;
+	default:
+		break;
+	}
+}
+
+//---------------------------------------------------------
+void FSteam::OpenAddGame()
+{
+	//adding a new game
+	FGame Game = AddANewGame();
+
+	//Menu to select if you want to add the game to a existing category or not
+	std::system("cls");
+	std::cout << "Would you like to add a game to an existing category?" << std::endl;
+	std::cout << "1 - Yes, please" << std::endl;
+	std::cout << "2 - No, Thanks" << std::endl;
+	std::cout << "Please choose an option to follow" << std::endl;
+	int UserInput;
+	std::cin >> UserInput;
+	
+	// adding the new game tocategoriy
+	AddNewGameToCategory(Game, UserInput);
+
+	//back to MainMenu
+	std::cin.ignore(500, '\n');
+	std::cout << "------------------------------------------\n" << std::endl;
+	std::cout << "Press enter to come back to Main menu" << std::endl;
+	if (std::cin.peek() == '\n')
+	{
+		ActiveMenuCommand = EMenuCommand::MainMenu;
+	}
+};
+
+//------------------------------------------------------
+void FSteam::OpenCategoryManager()
+{
+	std::system("cls");
+	std::cout << "-----------CATEGORY MANAGER-------------" << std::endl;
+	std::cout << "----------------------------------------" << std::endl;
+	std::cout << "1 - Create a new category" << std::endl;
+	std::cout << "2 - Delete a category" << std::endl;
+	std::cout << "3 - Main manu" << std::endl;
+	std::cout << "----------------------------------------" << std::endl;
+	std::cout << "choose an option" << std::endl;
+
+	int UserInput;
+	std::cin >> UserInput;
+	InputValidate(UserInput, 0, 4);
+
+	if (UserInput == 1)
+	{
+		if (AddNewCategory())
+		{
+			std::cout << "The category was added succesfully!" << std::endl;
+		}
+		else
+		{
+			std::cout << "something is wrong, the category wasn't be created..." << std::endl;
+		}
+
+		//back to MainMenu
+		std::cout << "Press enter to come back to Main menu" << std::endl;
+		if (std::cin.peek() == '\n')
```

Comment:
Same here as before. Every time you're doing this "Press enter to..." you're neglecting to handle any case in which the user doesn't press enter.